packages/module/src/FileDetailsLabel/FileDetailsLabel.tsx

```diff
@@ -49,7 +49,7 @@ export const FileDetailsLabel = ({
       </FlexItem>
       {isLoading && (
         <FlexItem>
-          <Spinner size="sm" />
+          <Spinner data-testid="spinner" size="sm" />
```

Comment:
Same question here. I'm not sure if hardcoding test ids is best practice.
If you agree, there are a few other places we'd want to remove the hardcoded ids